acceptance/user_managment_test.go

```diff
@@ -101,6 +101,38 @@ var _ = Describe("UserManagement", func() {
 		})
 	})
 
+	It("should allow an org manager to change a space users pemissions", func() {
+		var userRow *agouti.Selection
+
+		By("allowing the user to navigate to the space users page", func() {
+			Expect(page.Navigate(fmt.Sprintf(testEnvVars.Hostname+
```

Comment:
Mixing of concatenation with string specifiers
`fmt.Sprintf("%s/#/org/%s/spaces/%s/users", testEnvVars.Hostname, testOrg, testSpace)`